src/Control/Lens/TH.hs

```diff
@@ -99,10 +99,12 @@ import Control.Lens.Type () -- haddocks
 import Data.Char (toLower, isUpper)
 import Data.Foldable hiding (concat, any)
 import Data.List as List
-import Data.Map as Map hiding (toList,map,filter)
+import qualified Data.Map as Map hiding (toList,map,filter)
```

Comment:
Qualified imports only need `hiding` clauses when importing multiple modules `as` the same name.